File: IOS_TP1_1/IOS_TP1_1/ViewController.m
//
//  ViewController.m
//  IOS_TP1_1
//
//  Created by Yanwenli on 25/01/2019.
//  Copyright © 2019 Yanwenli_LOG_C_TP. All rights reserved.
//

#import "ViewController.h"
#import "HelloWorld.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    HelloWorld * helloword = [[HelloWorld alloc] init];
    helloword.text = @"Hi !";
    [helloword printHelloWorld];
    [helloword printHello:@"yes" :@"ok"];
    
    _label1.text = @"YANWENLI";
    _label2.text = @"polytech";
//    _label1.textColor = @"##12314";
    
     [_label1 setFont:[UIFont boldSystemFontOfSize:14]];
     [_label1 sizeToFit];
    // Do any additional setup after loading the view, typically from a nib.
}


@end

File: TP_ios/TP2_UserLoginAndRegistration/UserLoginAndRegistration/ViewController.swift
//
//  ViewController.swift
//  UserLoginAndRegistration
//
//  Created by Yanwenli on 12/02/2019.
//  Copyright © 2019 Yanwenli_ios_TP. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func viewDidAppear(_ animated: Bool) {
        let isUserLoggedIn = UserDefaults.standard.bool(forKey: "isUserLoggedIn");
        if(!isUserLoggedIn){
            self.performSegue(withIdentifier: "loginView", sender: self);


        }
    }
    
    
    @IBAction func logoutButtonTapped(_ sender: Any) {
        UserDefaults.standard.set(false, forKey: "isUserLoggedIn");
        UserDefaults.standard.synchronize();
        self.performSegue(withIdentifier: "loginView", sender: self);
    }
    

}

